e2e/test/scenarios/metrics/metrics-creation.cy.spec.ts

```diff
@@ -0,0 +1,254 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import type { StructuredQuestionDetails } from "e2e/support/helpers";
+import {
+  createQuestion,
+  echartsContainer,
+  enterCustomColumnDetails,
+  getNotebookStep,
+  modal,
+  popover,
+  restore,
+} from "e2e/support/helpers";
+
+const { ORDERS_ID, PRODUCTS_ID } = SAMPLE_DATABASE;
+
+type QuestionDetails = StructuredQuestionDetails & { name: string };
+
+const ORDER_COUNT_DETAILS: QuestionDetails = {
+  name: "Orders metric",
+  type: "metric",
+  query: {
+    "source-table": ORDERS_ID,
+    aggregation: [["count"]],
+  },
+  display: "scalar",
+};
+
+const PRODUCT_COUNT_DETAILS: QuestionDetails = {
+  name: "Products metric",
+  type: "metric",
+  query: {
+    "source-table": PRODUCTS_ID,
+    aggregation: [["count"]],
+  },
+  display: "scalar",
+};
+
+describe("scenarios > metrics", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+    cy.intercept("POST", "/api/dataset").as("dataset");
+    cy.intercept("POST", "/api/card").as("createCard");
```

Comment:
I think it makes more sense if this intercept lives in the `saveMetric` helper where we wait for it?